pkgs/by-name/rl/rlama/package.nix

```diff
@@ -37,25 +38,41 @@ buildGoModule rec {
     makeWrapper
   ];
 
+  # Run only unit tests for core packages; skip e2e tests that require Ollama
+  checkPhase = ''
+    go test -v ./internal/domain/... ./pkg/vector/... ./internal/repository/...
+  '';
+
   postInstall = ''
     wrapProgram $out/bin/rlama \
```

Comment:
Perhaps you can switch to makeBinaryWrapper?

Reply:
Yes i can.